src/Examples/UnitModels/Reactors/gibbs_reactor_testing.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "\n",
+    "![](msr_flowsheet.png)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Importing required Pyomo and IDAES components\n",
+    "\n",
+    "\n",
+    "To construct a flowsheet, we will need several components from the pyomo and idaes package. Let us first import the following components from Pyomo:\n",
+    "- Constraint (to write constraints)\n",
+    "- Var (to declare variables)\n",
+    "- ConcreteModel (to create the concrete model object)\n",
+    "- Expression (to evaluate values as a function of variables defined in the model)\n",
+    "- Objective (to define an objective function for optimization)\n",
+    "- TransformationFactory (to apply certain transformations)\n",
+    "- Arc (to connect two unit models)\n",
+    "\n",
+    "For further details on these components, please refer to the pyomo documentation: https://pyomo.readthedocs.io/en/latest/\n",
+    "\n",
+    "From idaes, we will be needing the FlowsheetBlock and the following unit models:\n",
+    "- Mixer\n",
+    "- Compressor\n",
+    "- Heater\n",
+    "- GibbsReactor\n",
+    "\n",
+    "We will also be needing some utility tools to put together the flowsheet and calculate the degrees of freedom, tools for model expressions and calling variable values, and built-in functions to define property packages, add unit containers to objects and define our initialization scheme.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyomo.environ import (Constraint,\n",
+    "                           Var,\n",
+    "                           ConcreteModel,\n",
+    "                           Expression,\n",
+    "                           Objective,\n",
+    "                           TransformationFactory,\n",
+    "                           value,\n",
+    "                           units as pyunits)\n",
+    "from pyomo.network import Arc\n",
+    "\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "from idaes.models.properties.modular_properties.base.generic_property import (\n",
+    "        GenericParameterBlock)\n",
+    "from idaes.models.unit_models import (Mixer,\n",
+    "                                      Compressor,\n",
+    "                                      Heater,\n",
+    "                                      GibbsReactor)\n",
+    "\n",
+    "from idaes.core.util import get_solver\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "from idaes.core.util.initialization import propagate_state"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Importing required thermophysical package\n",
```

Comment:
Way too much of this notebook is just copy-and-pasted from the other notebook. Even if you were to correct the places it still says "equilibrium reactor", there's no need to have the same text in two different places.